marketplace/deployer_util/provision.py

```diff
@@ -671,9 +671,11 @@ def provision_storage_class(schema, prop, app_name, namespace, provisioner):
   if not provisioner:
     provisioner = _DEFAULT_STORAGE_CLASS_PROVISIONER
 
+  volume_binding_mode = 'Immediate'
```

Comment:
Totally on board with making changes as non-invasive as possible. Though wondering if we have any reason to believe that the vsphere-volume wouldn't handle `WaitForFirstConsumer`? This branch might be hard to find later if we're trying to debug an Anthos-specific failure.

Reply:
This isn't supported for vsphere, will link to docs in code